### PR TITLE
Import python/alembic from userland-gate

### PR DESCRIPTION
**Pull request by 0xffea**

### CODE REVIEW

File: components/python/alembic/alembic-PYVER.p5m [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2013, 2015, Oracle and/or its affiliates. All rights reserved.
+#
+
+set name=pkg.fmri \
+    value=pkg:/library/python/alembic-$(PYV)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
```
Comment (pyhalov) [inline components/python/alembic/alembic-PYVER.p5m:27]: we haven't moved to this naming scheme yet, we still use library/python-2/*

File: components/python/alembic/alembic-PYVER.p5m [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+file path=usr/lib/python$(PYVER)/vendor-packages/alembic/templates/multidb/script.py.mako
+file path=usr/lib/python$(PYVER)/vendor-packages/alembic/templates/pylons/README
+file path=usr/lib/python$(PYVER)/vendor-packages/alembic/templates/pylons/alembic.ini.mako
+file path=usr/lib/python$(PYVER)/vendor-packages/alembic/templates/pylons/env.py
+file path=usr/lib/python$(PYVER)/vendor-packages/alembic/templates/pylons/script.py.mako
+file path=usr/lib/python$(PYVER)/vendor-packages/alembic/util.py
+#
+license alembic.license license=MIT
+
+# force the rename with an optional dependency on the old name
```
Comment (pyhalov) [inline components/python/alembic/alembic-PYVER.p5m:93]: so this dependency is also not necessary

File: components/python/alembic/history [hunk 3]
```diff
@@ -0,0 +1,4 @@
+library/python-2/alembic-26@0.7.4-5.12.0.0.0.86.0
```
Comment (pyhalov) [inline components/python/alembic/history:1]: I'm not sure that oi-userland handles these history files

File: components/python/alembic/Makefile [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+COMPONENT_TEST_ARGS=		setup.py test
+
+# common targets
+build:		$(BUILD_NO_ARCH)
+
+install:	$(INSTALL_NO_ARCH)
+
+test:		$(TEST_NO_ARCH)
+
+system-test:    $(SYSTEM_TESTS_NOT_IMPLEMENTED)
```
Comment (pyhalov) [inline components/python/alembic/Makefile:60]: system-test target is not supported.
REQUIRED_PACKAGES are also ignored, please, remove these lines

**Comment by xen0l** [comment]
@0xffea any updates?

**Pull request closed without merging**